packages/@tinacms/fields/src/ColorPicker/ColorPicker.tsx

```diff
@@ -131,6 +132,8 @@ export const Cover = styled.div`
 
 interface Props {
   colorFormat: ColorFormat
+  userColors: string[]
+  widget: string
```

Comment:
The `widget` property is optional, and we can be more specific about which strings are acceptable

```suggestion
  widget?: "sketch" | "block"
```